storage-proofs/core/src/settings.rs

```diff
@@ -21,7 +21,7 @@ pub struct Settings {
     pub column_write_batch_size: u32,
     pub use_gpu_tree_builder: bool,
     pub max_gpu_tree_batch_size: u32,
-    pub rows_to_discard: u32,
+    pub oct_rows_to_discard: u32,
```

Comment:
I think we don't need to expose the arity here, since we use oct everywhere we discard tree rows.